spec/models/order_balance_spec.rb

```diff
@@ -45,7 +45,9 @@
     end
 
     it 'returns the balance wraped in a Money object' do
-      expect(order_balance.display_amount).to eq(Spree::Money.new(20, currency: ENV['currency']))
+      expect(order_balance.display_amount).to eq(Spree::Money.new(20,
+                                                                  currency: ENV.fetch('currency',
+                                                                                      nil)))
```

Comment:
I was curious about this rule to [looked it up](https://stackoverflow.com/questions/47985660/rails-difference-between-env-fetch-and-env) to see why. 

> `fetch` allows you to specify the behavior if a key is not found

Which can be to return a default value, or raise an exception.

I guess this is intended to remind the developer that they should consider those options. In fact, I think the latter would be more appropriate in this and the preceding spec. 
I don't think the message makes that option clear:

> Style/FetchEnvVar: Use ENV.fetch('DISCOURSE_URL') or ENV.fetch('DISCOURSE_URL', nil) instead of ENV['DISCOURSE_URL'].

So to me, this just makes the code unnecessarily longer without improving quality. 

But.. I don't think we need to debate it, let's just accept the standard and stick with it ;)